src/C/cpl_vsi.jl

```diff
@@ -75,60 +66,53 @@ function VSIFWrite()
     ccall((:VSIFWrite, libgdal), Cint, ())
 end
 
-
 """
     VSIFGets(char * pszBuffer,
              int nBufferSize,
              FILE * fp) -> char *
 """
-function VSIFGets(arg1, arg2::Cint, arg3)
-    ccall((:VSIFGets, libgdal), Cstring, (Cstring, Cint, Ptr{FILE}), arg1, arg2, arg3)
+function VSIFGets(arg1, arg2, arg3)
+    ccall((:VSIFGets, libgdal), Cstring, (Cstring, Cint, Ptr{Cint}), arg1, arg2, arg3)
```

Comment:
Is it expected for `Ptr{FILE}` to be replaced by `Ptr{Cint}`?

Reply:
I think so, yes, due to Clang.jl changes. `FILE` is no longer being wrapped and put into common.jl either.